code/game/machinery/rechargestation.dm

```diff
@@ -183,15 +191,21 @@
 
 
 /obj/structure/machinery/recharge_station/proc/go_out()
-	if(!( src.occupant ))
+	if(!occupant)
 		return
-	//for(var/obj/O in src)
-	// O.forceMove(src.loc)
-	if (src.occupant.client)
-		src.occupant.client.eye = src.occupant.client.mob
-		src.occupant.client.perspective = MOB_PERSPECTIVE
-	src.occupant.forceMove(loc)
-	src.occupant = null
+	var/mob/living/cyborg = occupant
+
+	if(cyborg.client)
+		cyborg.client.eye = cyborg.client.mob
+		cyborg.client.perspective = MOB_PERSPECTIVE
+
+	cyborg.forceMove(loc)
+	if(exit_stun)
+		cyborg.apply_effect(exit_stun, STUN) //Action delay when going out of a closet
```

Comment:
After #4842, calling the status procs directly is preferred over `apply_effect()`:
https://github.com/cmss13-devs/cmss13/blob/12329b6a4ab55c628ed11145ff60da64d4d91b4d/code/modules/mob/living/damage_procs.dm#L72-L73
```suggestion
	if(exit_stun)
		cyborg.Stun(exit_stun) //Action delay when going out of a closet
```